app/helpers/application_helper.rb

```diff
@@ -214,20 +210,20 @@ def link_to_help(help_entry, link = '<span class="symbol question"><span>?</span
   #
   # You can also have multiple possible flash alerts in a single location with:
   #       <%= flash_div :error, :caution, :notice %>
-  # (These are the three varieties currently defined.) 
+  # (These are the three varieties currently defined.)
   #
   def flash_div *keys
-    keys.collect { |key| 
+    keys.collect { |key|
       if flash[key]
         if flash[key].is_a?(Array)
-          content_tag(:div, content_tag(:ul, flash[key].map {|flash_item| content_tag(:li, h(flash_item))}.join("\n").html_safe), :class => "flash #{key}") 
+          content_tag(:div, content_tag(:ul, flash[key].map {|flash_item| content_tag(:li, h(flash_item))}.join("\n").html_safe), :class => "flash #{key}")
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space between { and | missing.<br>Space missing inside }.